pkg/shell/shell.go

```diff
@@ -188,7 +182,7 @@ func (sp ShellProxy) ConnectFunc(w http.ResponseWriter, r *http.Request) {
 		io.Copy(pip, stdin)
 	}()
 
-	err = sess.RequestPty("xterm", 40, width, ssh.TerminalModes{ssh.ECHO: 1, ssh.TTY_OP_ISPEED: 14400, ssh.TTY_OP_OSPEED: 14400})
+	err = sess.RequestPty("xterm", 40, 80, ssh.TerminalModes{ssh.ECHO: 1, ssh.TTY_OP_ISPEED: 14400, ssh.TTY_OP_OSPEED: 14400})
```

Comment:
Is the general idea here that we start out a session at `40x80` and then `SIGWINCH` it to the correct size? Why not create the terminal session at the initial size we know we want, then only on resize change it via `SIGWINCH`?

Reply:
Yep -- that's exactly the idea. I think I got confused and thought `width` was on my branch (I didn't realize the UI was already sending that data). The corresponding UI code for this is sending a resize signal on initialization but, to your point, we can remove that step and set the correct size the first time.